public/app/plugins/panel/graph3/GraphPanel.tsx

```diff
@@ -188,7 +188,7 @@ export const GraphPanel: React.FC<GraphPanelProps> = ({
     }
     if (seriesGeometry.length > 1) {
       geometries.push(
-        <SeriesGeometry key={`seriesGeometry-${scale}-${i}`} scaleKey={scale}>
+        <SeriesGeometry key={`seriesGeometry-${scale}-${i}`} scaleKey={scale} nullValues={customConfig?.nullValues}>
```

Comment:
This is going to work only if you have more than one geometry(line+points, points+area etc). If you config to display the Line only then this setting will not be applied. I think we need to extend the Line and Point geometry interface to accept property `nullValues` so that the span gaps of the line can be controlled independently from the SeriesGeometry as well :)

Reply:
ah, good point!

Reply:
Would it be any drawbacks of always adding a SeriesGeometry group regardless of how many series there are?

Reply:
No, not really. But since the graph primitives are to be used by plugin devs as well, I thought it would be nice to have possibility to make the composition as plain as possible.

But when I think about it it actually adds unnecessary complexity. I think you suggestion makes sense to always require wrapping geomerties in SeriesGeometry - let's do it!

Reply:
The null values mode is already on each field...  can we use it from there?
https://github.com/grafana/grafana/blob/master/packages/grafana-data/src/types/dataFrame.ts#L58

Reply:
Great feedback, will have a look!